Dockerfile-alpine

```diff
@@ -85,17 +84,20 @@ RUN apk update && \
     sed -i 's/extension\s\?=/;extension =/' $CONF_PHPMODS/00_pgsql.ini && \
     /bin/bash -e /clean.sh
 
-# Install Alpine-compatible NewRelic, seed with variables to be replaced
+# Locate and install latest Alpine-compatible NewRelic, seed with variables to be replaced
 # Requires PHP to already be installed
-RUN curl -L https://download.newrelic.com/php_agent/archive/${NEWRELIC_VERSION}/newrelic-php5-${NEWRELIC_VERSION}-linux-musl.tar.gz -o /root/newrelic-php5-${NEWRELIC_VERSION}-linux-musl.tar.gz && \
+RUN NEWRELIC_MUSL_PATH=$(curl https://download.newrelic.com/php_agent/release/ 2>&1 | grep 'linux-musl.tar.gz' | cut -d '"' -f2) && \
```

Comment:
This line is a bit fragile, but I don't really see a better way to do this offhand.  curl should be `curl -s URL` instead of `curl URL 2>&1`